_pages/dev/server-side-rendering-in-spartacus.md

```diff
@@ -333,5 +333,5 @@ If you are involved in Spartacus internal development, or wish to submit a pull
 1. Start Spartacus with the SSR server by running the following command:
 
     ```bash
-    yarn start:ssr
+    yarn serve:ssr
```

Comment:
Do we need this documentation since we haven't yet implemented SSR on our side?

Reply:
It's change from develop branch, it will disappear after we merge develop into fsa branch.